src/test/java/com/coherent/task40/WaitUserTest.java

```diff
@@ -0,0 +1,41 @@
+package com.coherent.task40;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static com.coherent.task40.TestConstants.*;
+
+public class WaitUserTest {
+    WebDriver driver = new ChromeDriver();
+    By userImage = By.xpath("//div[@id='loading']/img");
```

Comment:
According to code style variable created on class level should have access modifier.  Using default access modifier doesn't looks right, as for most of them **private** will be enough
Please review all classes in the PR